google_gke_infra/README.md

```diff
@@ -1,56 +1,58 @@
 # GKE Infra
 
-This Terraform module provisions a regional `GKE cluster`, `VPC`, and `Subnet`. Optionally, it can be configured to create a service account with limited permissions for the K8s Nodes if no `service_account` value is provided (see [Use Least Privilege Service Accounts for your Nodes](https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster#use_least_privilege_sa)).
+This Terraform module provisions a regional Google Kubernetes Engine (GKE) cluster, Virtual Private Cloud (VPC), and Subnet. Optionally, you can configure it to create a service account with limited permissions for the Kubernetes Nodes if no value is provided for the **service_account**. For details, see [Use least privilege service accounts for your Nodes](https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster#use_least_privilege_sa).
+
+Use the following links to navigate through the document.
 
 - [GKE Infra](#gke-infra)
   - [Prerequisites](#prerequisites)
   - [Usage](#usage)
-    - [Basic Usage Example](#basic-usage-example)
-    - [Creating a Private Cluster](#creating-a-private-cluster)
-    - [Using the Cluster as a TF Provider](#using-the-cluster-as-a-tf-provider)
-    - [Deploying Custom K8s Resources](#deploying-custom-k8s-resources)
-    - [Upgrading a Cluster](#upgrading-a-cluster)
-      - [Update `k8s_version`](#update-k8sversion)
-      - [Update `node_version`](#update-nodeversion)
+    - [Basic configuration](#basic-configuration)
+    - [Create a private cluster](#create-a-private-cluster)
+    - [Use the cluster as a Terraform provider](#use-the-cluster-as-a-terraform-provider)
+    - [Deploy custom Kubernets resources](#deploy-custom-kubernetes-resources)
+    - [Upgrade a cluster](#upgrade-a-cluster)
+      - [Update **k8s_version**](#update-k8sversion)
```

Comment:
Variables should be written in bold: https://kyma-project.io/community/guidelines/content/#formatting-formatting-use-bold-font-for-these-items